src/Fade.ts

```diff
@@ -71,7 +71,9 @@ class Fade implements Plugin {
       const target = selector ? el.querySelector<HTMLElement>(selector)! : el;
       const opacity = Math.min(1, Math.max(0, (1 - Math.abs(progress * scale))));
 
-      target.style.opacity = `${opacity}`;
+      if(target) {
```

Comment:
It's missing space between `if` and the parenthesis, looks like the linter couldn't catch it.

Reply:
fixed